Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/KeyView.java

```diff
@@ -157,7 +157,8 @@ public boolean isEditable() {
     @Override
     public boolean isFilterable() {
         InputType inputType = getInputType();
-        if (InputType.MULTIPLE_SELECTION.equals(inputType) || InputType.ONE_LINE_SINGLE_SELECTION.equals(inputType)) {
+        if (InputType.MULTIPLE_SELECTION.equals(inputType) || InputType.MULTI_LINE_SINGLE_SELECTION.equals(
```

Comment:
Sorry, I think this edit doesn't make sense because `MULTI_LINE_SINGLE_SELECTION` maps to an input type `selectOneRadio` which cannot be filtered (`selectOneRadio` does not support the "filter" and "filterMatchMode" attributes).

```
case MULTI_LINE_SINGLE_SELECTION:
                return "oneRadio";
```

Reply:
Yes, there was a reason. I vaguely remember checking on that. During the review, I didn't find it right away, and I assumed I had simply forgotten it.